### PR TITLE
Monitoring merge

### PR DESCRIPTION
**Pull request by cbosdo**
Merge Joao's scripts to setup grafana demo with the existing grafana module.

### CODE REVIEW

**Comment by MalloZup** [comment]
I have beeing using this somehow from my parallel project. sofar looks good to me. Don't stop. but wait for feedback :grin:

File: salt/grafana/init.sls [hunk 1]
```diff
@@ -65,7 +69,7 @@ grafana:
 grafana_anonymous_login_configuration:
   file.blockreplace:
     - name: /etc/grafana/grafana.ini
-    - marker_start: '#################################### Anonymous Auth ##########################'
+    - marker_start: '#################################### Anonymous Auth ######################'
```
Comment (MalloZup) [inline salt/grafana/init.sls]: this was broken corect

File: salt/grafana/prometheus/alert-rules.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+groups:
```
Comment (MalloZup) [inline salt/grafana/prometheus/alert-rules.yml]: `alert-rules`  isn't the canonical name. you should call it `rules.yaml` following upstream doc

File: salt/grafana/prometheus/alert-rules.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  - alert: InstanceDown
+    expr: up == 0
+    for: 10s
+    labels:
+      severity: critical
+    annotations:
+      summary: "Instance DOWN"
+      description: "{{ '{{' }} $labels.instance }} has been down for more than 1 minute"
+
+# Load alerts
```
Comment (MalloZup) [inline salt/grafana/prometheus/alert-rules.yml]: remove comments if not used, dead code

File: salt/grafana/init.sls [hunk 4]
```diff
@@ -8,53 +8,57 @@ prometheus:
...
+    - enable: True
 
-        [Install]
-        WantedBy=multi-user.target
-    - require:
-      - pkg: prometheus
+prometheus-suma_sd:
+  pkg.installed:
+    - names:
+      - golang-github-cavalheiro-prometheus-suma_sd
```
Comment (MalloZup) [inline salt/grafana/init.sls]: seems a bit of hack here

File: salt/grafana/prometheus/alertmanager.yml [hunk 5]
```diff
@@ -0,0 +1,71 @@
+global:
+  # The smarthost and SMTP sender used for mail notifications.
+  smtp_smarthost: 'relay.suse.de:25'
+  smtp_from: 'monitoring-susecon@suse.de'
```
Comment (MalloZup) [inline salt/grafana/prometheus/alertmanager.yml]: seeems really a strange adress

**Comment by cbosdo** [comment]
Closing: too much bit rotten and may be not relevant anymore

**Comment by moio** [comment]
Sorry @cbosdo, my fault not having been able to follow up with this in a timely manner. Hope to resume discussions at some later point.

Thanks anyway

**Pull request closed without merging**